scripts/west_commands/runners/native_gdb.py

```diff
@@ -36,11 +36,11 @@ def do_run(self, command: str, **kwargs):
         if self.cfg.gdb is None:
             raise ValueError("The provided RunnerConfig is missing the required field 'gdb'.")
 
-        if self.cfg.elf_file is None:
-            raise ValueError("The provided RunnerConfig is missing the required field 'elf_file'.")
+        if self.cfg.exe_file is None:
+            raise ValueError("The provided RunnerConfig is missing the required field 'exe_file'.")
```

Comment:
This doesn't seem right, how does someone use `ninja debug` with an actual application for a real board?

Reply:
`debug` is handled by the other runners, not an issue